our-work/collaborations/mcdonald.yml

```diff
@@ -1,8 +1,8 @@
 title: McDonald Lab
 id: mcdonald-lab
 description: Dr. McDonald's primary research focus is on improving maternal and infant health in resource poor settings, with particular emphasis on the role of the placenta in mediating the impact of maternal infectious disease on fetal development. CCV is collaborating with Dr. McDonald on applying metagenomics methods to study the impact of schistosomiasis on the breast milk and placental microbiome.
-tags:
-  - active
+active: True
+department:
```

Comment:
@ashokrags we are separating tags from department. Would you advise on what to have here on each o those categories?